script_runner/tests/test_ctx_wrapper.py

```diff
@@ -45,8 +60,6 @@ def cleanup(self):
     def _prescript(self):
         return (
             '#!/usr/bin/env python\n'
-            'import sys\n'
-            'sys.path.append("{0}")\n'
```

Comment:
instead of manipulating sys.path in this script, it's better to set the PYTHONPATH env variable, because that doesn't have the problem of escaping the path when putting it in a python script